src/elements/FlexRow/FlexRow.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { getBreakpoint, numberLikeProp } from '../../util/helpers';
+import Common from '../Common/Common';
+
+const FlexRow = ({ children, breakMedium, breakSmall, breakCustom, gutter = 'none', ...other }) => {
+	const combinedClasses = classNames(
+			'ui-flex-row',
+			`gutter-${gutter}`,
+			other.classes
+		),
+		breakpoint = getBreakpoint(),
+		style = {};
+
+	switch (breakpoint) {
```

Comment:
Wouldn't this block be simpler as a one liner? 

`if(breakpoint !== 'Large' && breakMedium || breakSmall) style.display = 'block'`

Reply:
No, because that means if the breakpoint was `Medium`, but you had a `breakSmall` prop, it would still break because the condition would be true.

Reply:
https://media0.giphy.com/media/xT5LMzIK1AdZJ4cYW4/giphy.gif